_schemaV2.graphql

```diff
@@ -12132,6 +12132,62 @@ type MarketingCollectionQuery {
   tagID: String
 }
 
+enum MarketingCollectionsSorts {
+  CREATED_AT_ASC
+  CREATED_AT_DESC
+  UPDATED_AT_ASC
+  UPDATED_AT_DESC
+}
+
+type MarketingCollectionType implements Node {
```

Comment:
this shouldn't be `MarketingCollectionType`, but just `MarketingCollection` (the `Type` isn't a good naming).

if this conflicts with the stitched-in version name, I would potentially recommend an Unleash flag to determine whether the stitched-in version is being used or not